auth0/structure_auth0_connection.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func flattenConnectionOptions(d ResourceData, options interface{}) []interface{} {
-	if options == nil || optionsBlockIsNotDefined(d) {
```

Comment:
This is where Issue #33 gets fixed. 

After checking the terraform plugin SDK docs over [here](https://www.terraform.io/plugin/sdkv2/best-practices/detecting-drift), even though the options might be omitted from the terraform config we should always record it within the state in the READ function as the Auth0 API assigns a couple of default values to certain elements. 

This is also the reason why importing failed to get the options assigned within the state, although it was working fine after creating a new resource.